docs/en/coresnaps.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Ubuntu IoT Developer Documentation
+---
+
+# Snaps in Ubuntu Core
+
+Ubuntu Core is built from _snaps_, a secure, confined, dependency-free,
+cross-platform Linux packaging format.
+
+Snaps are entirely self-contained, even to the point of encapsulating their own
+file system, and this means they include everything needed to run. They're used
+by Ubuntu Core to both compose the image that's run on a device, and to deliver
+consistent and reliable software updates, often to low-powered, inaccessible,
+and remotely administered embedded and IoT systems.
+
+Whether it's an update to a single device, a specific subset of devices, or a
+deployment of tens of thousands of devices, snaps enable Ubuntu Core to
+maintain and verify a system’s integrity:
+
+- **Autonomous updates to any device** - the update service needs to be a
+  reliable and automatic process, catering for a predictable update cadence and
+an optional fine level of control over when and how updates are delivered
+
+- **Incomplete and problematic update recovery** - the inaccessible nature of
+  many embedded device installations makes it imperative that updates are
+impregnable and capable of withstanding broken, blocked, partial and
+interrupted updates
+
+- **Critical update provision** - in specific circumstances when an ad-hoc or
+  critical update needs to be made, the update system needs to give these
+priority and incorporate the update back into the regular update service
+
+- **Unpredictable hardware and network conditions** - in situations that can’t
+  be easily  modelled or predicted, and update system needs to have enough
```

Comment:
there are a few double spaces here and there, if you care about that (markdown removes them anyhow)

Reply:
/and/any

Reply:
Thanks Nick! Most of the double spaces are to help with the list formatting, though I'm not sure if they still make it through to the final published site (I'll check).